recipe/meta.yaml

```diff
@@ -24,7 +23,7 @@ requirements:
   run:
     - python
     - numpy >1.20.0
-    - pytorch >=1.8.1
+    - pytorch >=1.10.0
     - lightning-utilities >=0.8.0
     - typing-extensions  # [py<39]
```

Comment:
Can you add run_constrained with reqs from audio.txt, detection.txt, image.txt, multimodal.txt, text.txt and visual.txt?
https://github.com/Lightning-AI/torchmetrics/tree/v1.4.0.post0/requirements
https://inspector.pypi.io/project/torchmetrics/1.4.0.post0/packages/6d/e6/e51997d1818a4c1a1ad2b1c7ca5ff9dd95969596add58b2ed39479026964/torchmetrics-1.4.0.post0-py3-none-any.whl/torchmetrics-1.4.0.post0.dist-info/METADATA#line.28

Reply:
Again, upper bounds aren't relevant, please remove

Reply:
Done. I left the upper bounds if there are no lower bounds

Reply:
just wondering what's the reasoning - in that case maybe there should just be no bounds at all?
[worth a glance over](https://iscinumpy.dev/post/bound-version-constraints/) - tl;dr is that upper bounds should be used sparingly and with consideration